metricbeat/module/postgresql/statement/statement.go

```diff
@@ -64,23 +67,27 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 // Fetch methods implements the data gathering and data conversion to the right
 // format. It publishes the event which is then forwarded to the output. In case
 // of an error set the Error field of mb.Event or simply call report.Error().
-func (m *MetricSet) Fetch() ([]common.MapStr, error) {
+func (m *MetricSet) Fetch(reporter mb.ReporterV2) {
 	db, err := sql.Open("postgres", m.HostData().URI)
 	if err != nil {
-		return nil, err
+		logger.Error(err)
```

Comment:
Hurts me to see this as we soon have https://github.com/elastic/beats/pull/10727 but it's better to get this in if we have it green and do a bulk update later.

Reply:
It hurts everyone :smile: but I thought that if we change them all and we try to make them look the same, maybe we can just later make a bulk replacement in one single PR

Reply:
yes, agree.